lib/kasket/configuration_mixin.rb

```diff
@@ -79,7 +79,16 @@ def has_kasket_on(*args)
       extend ReadMixin unless methods.map(&:to_sym).include?(:find_by_sql_with_kasket)
     end
 
+    def kasket_expires_in(time)
+      @kasket_expires_in = time
+    end
+
+    def _kasket_expires_in
```

Comment:
defined above. otherwise would probably just use `attr_accessor` seems nicer to be able to do `kasket_expires_in 5.minutes` rather than `kasket_expires_in = 5.minutes`